src/lib/components/side-nav.svelte

```diff
@@ -101,5 +102,22 @@
         data-testid="labs-mode-button"
       />
     </slot>
+    <slot name="feedback">
+      {#if !$useDarkMode}
+        <NavigationButton
+          onClick={() => ($useDarkMode = true)}
+          tooltip={translate('common.night')}
+          label={translate('common.night')}
+          icon="moon"
+        />
+      {:else}
+        <NavigationButton
+          onClick={() => ($useDarkMode = false)}
+          tooltip={translate('common.day')}
+          label={translate('common.day')}
+          icon="sun"
+        />
+      {/if}
```

Comment:
Looks like there's an unnecessary `<slot name="feedback">` in here that we'll want to remove and I think we want to add a `LabsModeGuard`. 

Also, what do you think about using ternaries in the comment props instead?

```suggestion
      <LabsModeGuard>
        <NavigationButton
          onClick={() => ($useDarkMode = !$useDarkMode)}
          tooltip={$useDarkMode
            ? translate('common.day')
            : translate('common.night')}
          label={$useDarkMode
            ? translate('common.day')
            : translate('common.night')}
          icon={$useDarkMode ? 'sun' : 'moon'}
        />
      </LabsModeGuard>
```

Reply:
This slot="feedback" we wanna keep. This way we don't need to worry about Cloud and can add Labs/Dark Mode to it's slot="feedback" when ready.